addons/project/static/src/views/project_task_kanban/project_task_kanban_view.scss

```diff
@@ -9,4 +9,24 @@
         padding-right: 3px;
         font-size: 16px;
     }
+
+    .o_kanban_record {
+        @include media-breakpoint-up(sm) {
+            &:hover {
+                .o_field_widget.o_field_many2many_tags_avatar .o_quick_assign {
+                    visibility: hidden;
```

Comment:
What does that do? Sorry, my css skills are not good

Reply:
we have overridden CSS here to first make it invisible on o_kanban_record and apply CSS separately on the task and sub-task list.